testsuites/CBLTester/System_test_multiple_device/conftest.py

```diff
@@ -283,6 +282,7 @@ def params_from_base_suite_setup(request):
             raise
 
     # Create CBL databases on all devices
+    sg_admin_url = cluster_topology["sync_gateways"][0]["admin"]
```

Comment:
is this a code change or just moved some lines down? the comment above this line doesn't sound matching the logic this line.

Reply:
actually there is no difference. Hemant just moved from one line to another

Reply:
Actually no, the reason I moved it because I was initialising the sg_admin_url, before if condition for sg_ssl flag. This was causing the issue for sg_client.create_user call. So basically, admin url get initialised to "http", but after checking for sg_ssl flag, the config file may or may not change for ssl value and thus in case of True, we still use http instead of https, leading to error.